katas/args/andres/parser.js

```diff
@@ -0,0 +1,47 @@
+import { Argument } from './argument';
+
+export class Parser {
+    constructor (newSchemaString) {
+        this.schemaArray = this.splitArgumentList(newSchemaString); // splits and filters the schema args string
+        this.flagArray = this.takeArgumentList(this.schemaArray); // stores an array of arguments
+
```

Comment:
Try to avoid empty lines in case those do not help to group lines of code inside functions